tcf_website/templates/browse/browse.html

```diff
@@ -12,7 +12,8 @@
         {% include "../common/notification.html" %}
     </div> {% endcomment %}
     <div class="browse container">
-        {% include "../common/review-drive-banner.html" %}
+        <!-- {% include "../common/review-drive-banner.html" %} -->
```

Comment:
Same for everywhere else: I'd recommend just removing lines instead of commenting them out, since we can look at commits to add them back in. Otherwise, we'll have a lot of comments everywhere. IMO, remove all the comments in the commit.